modules/lib/tsconfig.json

```diff
@@ -1,17 +1,46 @@
-{
+{ // This file will be used by the IDE.
     "compilerOptions": {
+        // NOTE: This is probably correct since we're still building for Nashorn.
         "lib": ["es5"],
+
+        // TODO: This only matters at buildtime, perhaps move to
+        // tsconfig.build.json
         "removeComments": false,
+
+        // The target setting changes which JS features are downleveled and
+        // which are left intact. Changing target also changes the default value
+        // of lib. You may “mix and match” target and lib settings as desired,
+        // but you could just set target for convenience.
+        // NOTE: This is probably correct since we're still building for Nashorn.
         "target": "es5",
+
+        // NOTE: This is probably correct since we're still building for Nashorn.
         "module": "commonjs",
+
+        // TODO: Maybe use "node16" or "bundler" since we're not using file
```

Comment:
Option `bundler` can only be used when `module` is set to `preserve` or to `es2015` or later. Option `module` must be set to `NodeNext` when option `moduleResolution` is set to `NodeNext`. Also, it's not like we are even targeting `NodeNext`, as the Node.js is not our target platform.
Just let's keep `node`.

Reply:
I believe it's about module resolution at build time, not in the produced target code.

node means node9 or older, which version of node are you using in the compile time environment again?...

https://www.typescriptlang.org/tsconfig/#moduleResolution

'node10' (previously called 'node') for Node.js versions older than v10, which only support CommonJS require. You probably won’t need to use node10 in modern code.

Reply:
Hmm, when trying node16 I get: Option 'module' must be set to 'Node16' when option 'moduleResolution' is set to 'Node16'.

Reply:
So I guess keeping "node", is the way for now.